ebpf/include/vmlinux/vmlinux_common.h

```diff
@@ -0,0 +1,11 @@
+#ifndef __VMLINUX_COMMON_H__
```

Comment:
what is the origin of this file? similar for the other headers below

Reply:
these header files are from https://github.com/xdp-project/bpf-examples/tree/master/headers/vmlinux with modification, the aim is to include minimum header files for the bpf program, not include full vmlinux.h header file which is too big